src/main/kotlin/racingCar/model/Cars.kt

```diff
@@ -1,13 +1,18 @@
 package racingCar.model
 
-class Cars(numberOfCars: Int, private val distanceOfDrive: () -> Int = { 0 }) {
+class Cars(stringOfCars: List<String>) {
 
-    var list = List(numberOfCars) { Car() }
+    private val list = stringOfCars.map { Car(it) }
 
-    fun move() {
-        list = list.map { moveCar(it) }
+    fun move(distanceOfDrive: () -> Int) {
+        list.map { moveCar(it, distanceOfDrive) }
     }
 
-    private fun moveCar(car: Car) = car.move(distanceOfDrive.invoke())
+    private fun moveCar(car: Car, distanceOfDrive: () -> Int) = car.move(distanceOfDrive.invoke())
 
+    fun result() = list.map { it.info() }
+
+    fun victoryPlayer() = list.map { it.info() }.filter { it.second == maxResult() }.map { it.first }
```

Comment:
확장함수를 여러개 붙여서 쓰는 경우에는 한 줄에 . 하나 씩 나열하는 것이 좋습니다.
만약 이어져 있는 함수 중 한 곳에서 exception이 발생하는 경우 로그에는 몇 번째 줄인지는 알려주지만, 어느 함수인지 알려주지는 않아서 디버깅하는데 어려워집니다. 🙂

```suggestion
    fun victoryPlayer() = list.map { it.info() }
        .filter { it.second == maxResult() }
        .map { it.first }
```

Reply:
리뷰어님!! 너무 감사합니다.😊

오늘 회사에서 위의 고치기 전의 코드와 비슷한 코드에서 에러가 나는 바람에 Exception 발생 지점에 대해 스택트레이스가 힘들었는데,

리뷰어님이 말해주신 방법을 적용하여, Exception 발생 지점을 좀 더 명확하게할 수 있었습니다.